src/client/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -118,6 +118,10 @@ export class KernelDebugAdapter implements DebugAdapter, IKernelDebugAdapter {
             if (content.event === 'stopped') {
                 this.runByLineThreadId = content.body.threadId;
                 this.runByLineSeq = content.seq;
+
+                if (this.isRunByLine) {
+                    this.RunByLineStackTrace();
```

Comment:
Not sure why this is called here..

Reply:
Some comments would help (one line comment)

Reply:
I know this is old code `  if (this.session.name.includes('RBL=')) {`
But would be good to have a constant instaed of hardcoding `RBL` i wouldn't have known that its used in this manner at all.
I commented elsewhere suggesting we not use `RBL`

Reply:
Instead of using `RBL` why don't we add a custom configuration property item into the debugger. Just like we use `__document`, that way we dont have to look for particular strings.

Reply:
done